libyul/backends/evm/StackLayoutGenerator.cpp

```diff
@@ -540,6 +583,47 @@ void StackLayoutGenerator::stitchConditionalJumps(CFG::BasicBlock const& _block)
 			},
 			[&](CFG::BasicBlock::FunctionReturn const&)	{},
 			[&](CFG::BasicBlock::Terminated const&) { },
+			[&](CFG::BasicBlock::Switch const& _switch)
+			{
+				/*Stack exitLayout = info.exitLayout;
```

Comment:
You will need to keep this commented in, this part is crucial.

The commented code actually looks pretty much correct, apart from the fact that (as commented elsewhere), you don't jump to the target if there is a default case here.